draft-ietf-jsonpath-base.md

```diff
@@ -1453,12 +1453,12 @@ LCALPHA             = %x61-7A  ; "a".."z"
 function-expr       = function-name "(" S [function-argument
                          *(S "," S function-argument)] S ")"
 function-argument   = literal /
+                      singular-query /
                       filter-query / ; (includes singular-query)
+                      logical-expr /
                       function-expr
 ~~~
 
-A function argument is a `filter-query` or a `comparable`.
-
 According to {{filter-selector}}, a `function-expr` is valid as a `filter-query`
 or a `comparable`.
```

Comment:
This statement is true, but it's not obvious how it ties in with the preceding ABNF.

Reply:
I don't think it (According to..)  is that useful, so I would delete it.

Reply:
Agreed.